src/commands/install.rs

```diff
@@ -25,6 +25,10 @@ pub struct Install {
     /// Install latest version
     #[clap(long, conflicts_with_all = &["version", "lts"])]
     pub latest: bool,
+
+    /// Do not display a progress bar
+    #[clap(long)]
+    pub no_progress: bool,
```

Comment:
I don't think clap has a way to add `--progress/--no-progress` to enable or disable a flag, just made a progress bar the default.

Reply:
fyi, I'm gonna change that before releasing to `--progress=never`, and will have `--progress=auto|always` so `auto` will be based on the log level <picture data-single-emoji=":blob_thumbs_up:" title=":blob_thumbs_up:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/blob_thumbs_up/f3fed0d24f5f9a1f.png" alt=":blob_thumbs_up:" align="absmiddle"></picture>